sdcm/cluster_k8s/__init__.py

```diff
@@ -611,19 +626,6 @@ def deploy_scylla_cluster(self, target_mgmt_agent_to_minio: bool = False,
         cpu_limit = int(cpu_limit)
         memory_limit = convert_memory_units_to_k8s_value(memory_limit)
 
-        if target_mgmt_agent_to_minio:
```

Comment:
I agree that better to use native resources than it's analogs, but replacing "minio" with native resources is not "fix", it is improvement. And, moreover, must be in separate change.

Reply:
Mgmt case never worked and never been tested on EKS, so it is a fix, not improvement.